### PR TITLE
added future readme

### PR DESCRIPTION
**Pull request by Tompoudurans**
this is the readme put in Issue https://github.com/Tompoudurans/data-explore/issues/1 thought the code does not has all of it features yet

### CODE REVIEW

**Comment by drvinceknight** [comment]
> this is the readme put in Issue #1 thought the code does not has all of it features yet

The README looks fine but I'd cherry pick this commit to the branches that implement the functionality.

**Comment by Tompoudurans** [comment]
I have rebased this branch

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## A tool to capture information about the general structure of tables within an sql database
+
+The tool computes:
+the number of records in a table of a given database (count), the mean value of all records in the table (mean) as well as their standard deviation (std), and their (0%, 25%, 50%, 75%, 100%) quartile
+
+### Installation
+
```
Comment (drvinceknight) [inline README.md:7]: Can we add some bullet points / sentences here about what software is required to follow the instructions below.

**Review by drvinceknight — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+source `<environment>`/bin/activate
+
+### Installation
+
+Open the terminal and enter the following commands:
+
+    $ git clone https://github.com/Tompoudurans/data-explore
+    $ cd data-explore
+    $ python setup.py develop
+
```
Comment (drvinceknight) [inline README.md:39]:
```suggestion
### Installation

#### Basic software requirements

To install, the following is assumed to be available:

- Python 3.7 or 3.8 (the software might work on other versions but it is only tested on 3.7 or 3.8).
- A recent version of git.

If those are not available please get in touch with the maintainer for alternative ways of installing.

#### Virtual environment

It is recommended to run the software using a Python virtual environment (this ensures that different dependencies are isolated).

To setup the environment, run the following in a command line tool:

    $ python -m venv data-explore

Once the virtual environment is created, it needs to be activated with the following commands:

On Windows, run:

    C:\> data-explore\Scripts\activate.bat

On unix (linux or OS X):

    source data-explore/bin/activate

#### Installation

The following commands will download and install the software

    $ git clone https://github.com/Tompoudurans/data-explore
    $ cd data-explore
    $ python setup.py develop


```

**Review by drvinceknight — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    source data-explore/bin/activate
+
+#### Installation
+
+The following commands will download and install the software
+
+    $ git clone https://github.com/Tompoudurans/data-explore
+    $ cd data-explore
+    $ python setup.py develop
+
```
Comment (drvinceknight) [inline README.md:48]:
```suggestion


To confirm installation run the following:

    $ python -m dataexplorer --help
```

**Review by drvinceknight — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+The `<log>` file  produced by this command will contain all the information about the general structure of the `<table>` held in <`database`>.
+
+### Test
+
+If the tool needs to be tested after installing, enter the following commands:
+
+  `$  python -m pip install pytest.` If pytest is not already installed
+
+  `$  python -m pytest -p no:logging`
```
Comment (drvinceknight) [inline README.md:69]:
```suggestion
The tool is tested using the Python testing framework purest. To install pytest:

    $ python -m pip install pytest
    
To run the tests:    

    $  python -m pytest -p no:logging

```

**Review by drvinceknight — CHANGES_REQUESTED** [review]
Hadn't realised your markdown syntax wasn't quite right here. Have fixed that as well in my suggestion. (The installation command you wrote would give an error.)

**Comment by Tompoudurans** [comment]
i feel i should remove 
`### Required softwares

To be able to install the software, it is necessary first to have a version of git (any version)
as well as a version of python 3.7 or 3.8 (this software has not been tested for further versions).
Also its is important that pip is installed.`

as there is `#### Basic software requirements`  later on in the read me file

File: README.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
+## A tool to capture information about the general structure of tables within an sql database
+
+The tool computes:
+the number of records in a table of a given database (count), the mean value of all records in the table (mean) as well as their standard deviation (std), and their (0%, 25%, 50%, 75%, 100%) quartile
+
+### Required softwares
+
+To be able to install the software, it is necessary first to have a version of git (any version)
+as well as a version of python 3.7 or 3.8 (this software has not been tested for further versions).
+Also its is important that pip is installed.
```
Comment (drvinceknight) [inline README.md:10]:
```suggestion
```

**Comment by drvinceknight** [comment]
> i feel i should remove

Yeah absolutely I thought that had been removed.

**Pull request merged**